patches/server/0005-Paper-config-files.patch

```diff
@@ -465,7 +465,7 @@ index 0000000000000000000000000000000000000000..45f1436cdd4b81b621ab71e4336c2aa6
 +
 +@SuppressWarnings({"CanBeFinal", "FieldCanBeLocal", "FieldMayBeFinal", "NotNullFieldNotInitialized", "InnerClassMayBeStatic"})
 +public class GlobalConfiguration extends ConfigurationPart {
-+    static final int CURRENT_VERSION = 28;
++    static final int CURRENT_VERSION = 29;
```

Comment:
Don't think we need to update the version here

Reply:
We don't yea. Version bumps are  only needed if we need a migration to be run on existing data.

Reply:
alright thanks!